src/Controls/tests/Core.UnitTests/LinearGradientBrushTests.cs

```diff
@@ -73,6 +74,25 @@ public void TestNullOrEmptyLinearGradientBrush()
 			Assert.False(Brush.IsNullOrEmpty(linearGradientBrush));
 		}
 
+		[Fact]
+		public void TestNullOrEmptyLinearGradientPaint()
+		{
+			LinearGradientBrush linearGradientBrush = new LinearGradientBrush
+			{
+				StartPoint = new Point(0, 0),
+				EndPoint = new Point(1, 0),
+				GradientStops = new GradientStopCollection
+				{
+					new GradientStop(),
+					new GradientStop()
```

Comment:
What happens when one is null and the other is a color?

What happens if there is a null stop in the middle?